kernel/safe_typing.ml

```diff
@@ -336,8 +335,8 @@ type side_effect = {
   seff_body : (Constr.t, Vmemitcodes.body_code option) Declarations.pconstant_body;
   seff_univs : Univ.ContextSet.t;
 }
-(* Invariant: For any senv, if [Certificate.check senv seff_certif] then
-  senv where univs := Certificate.universes seff_certif] +
+(* Invariant: For any senv, if [Certificate.safe_extend senv seff_certif] returns [Some certif'] then
+  senv where univs := Certificate.universes certif'] +
```

Comment:
That's not quite true now, because the universe constraints could be inconsistent.

Reply:
reworded